composer.json

```diff
@@ -32,7 +32,7 @@
     ],
     "require": {
         "php": ">= 7.4",
-        "loophp/iterators": "^1.5.13"
+        "loophp/iterators": "^1.5.14"
```

Comment:
do we need to mention the minor version specifically or can we have this as just:
`"loophp/iterators": "^1.5"`

Reply:
Yeah we could do that, it's the same.